kratos/python_scripts/compare_two_files_check_process.py

```diff
@@ -360,6 +368,153 @@ def __CompareSolMetricFile(self):
         error /= nvertices
         self.assertTrue(error < self.tol, msg = self.info_msg)
 
+    def __CompareVtkFile(self):
```

Comment:
If we refactor this a little bit and move it to C, we can create a VTK IO to read a appart of writting (The submodelparts can be preserved with some auxiliar variable storing the "color") . just commenting

Reply:
my plan for the future is to move to vtu such that we don't have to write the mesh multiple times and also to use proper compression
In fact a friend of mine and me recently started with a small library to read/write vtu files
=> once this is ready I will create a VtuIO in Kratos based on this (will be able to read & write).

Reply:
I started a VTU library once too, but it depended of the official VTK library and it is a huge dependency, that would be standalone?

Reply:
ours does not have any dependencies (except c++11) and will be header-only, basically like pybind
=> he implemented this in python already, now we move it to c++ and add reading capabilities

Yep, the vtk-lib is huge, hence we want sth lightweight (if I think abt it, sounds like the same story with boost-python & pybind...)

Reply:
Nice, and indeed huge dependencies are bad

Reply:
my main motivation besides having vtu-support in the core is to use it for the FileIO in CoSimulation => i.e. when we couple to external solvers this will be the file format of choice (the big advantage/main reason to use it is that we can directly visualize it)